docs/libcurl/opts/CURLOPT_SSLVERSION.3

```diff
@@ -50,6 +50,18 @@ TLSv1.1 (Added in 7.34.0)
 TLSv1.2 (Added in 7.34.0)
 .IP CURL_SSLVERSION_TLSv1_3
 TLSv1.3 (Added in 7.52.0)
+.IP CURL_SSLVERSION_MAX_DEFAULT
+Use as flag with CURL_SSLVERSION_TLSv1_x and behavior is same as
+CURL_SSLVERSION_MAX_TLSv1_2.
```

Comment:
Even if NSS was the only library that currently allows one to set ```sslver.max``` to TLS library default, it is still worth to have this feature exposed via libcurl API.  We can hard-code it to TLSv1.2 for other libraries now and eventually implement propagation of the flag once other libraries introduce a similar API.

However, it needs to be hard-coded in libcurl implementation, not in its API/ABI.  If both ```CURL_SSLVERSION_MAX_DEFAULT``` and ```CURL_SSLVERSION_MAX_TLSv1_2``` were mapped to the same constant in libcurl API/ABI, it would be nearly impossible to change the default later on.